README.md

```diff
@@ -1,173 +1,106 @@
 
 ## Getting Started
 
 ### Prerequisites
-- Access to a Kubernetes v1.28+ cluster with supported NVIDIA GPUs
 
-### To Deploy on the cluster
+- Kubernetes v1.28 and higher.
+- NVIDIA GPUs that are supported by the NIM microservices to deploy.
+
+### Deploying the Operator on the Cluster
+
 **Build and push your image to the location specified by `IMG`:**
 
 ```sh
 make docker-build docker-push IMG=nvcr.io/nvidia/cloud-native/k8s-nim-operator:v0.1.0
 ```
 
-**NOTE:** This image ought to be published in the personal registry you specified.
-And it is required to have access to pull the image from the working environment.
-Make sure you have the proper permission to the registry if the above commands don’t work.
+> Publish the image to a personal registry.
+> You must be able to pull the image from the working environment.
+> Make sure you have the proper permission to the registry if the preceding commands result in an error.
 
 **Install the CRDs into the cluster:**
 
 ```sh
 make install
 ```
 
-**Deploy the Manager to the cluster with the image specified by `IMG`:**
+**Deploy the manager to the cluster with the image specified by `IMG`:**
 
 ```sh
-make deploy IMG=nvcr.io/nvidia/cloud-native/k8s-nim-operator:v0.1.0
+make deploy IMG=nvcr.io/nvstaging/cloud-native/k8s-nim-operator:tag
 ```
 
-> **NOTE**: If you encounter RBAC errors, you may need to grant yourself cluster-admin
+> If you encounter RBAC errors, you might need to grant yourself cluster-admin
 privileges or be logged in as admin.
+> Ensure that the samples have default values.
 
->**NOTE**: Ensure that the samples has default values to test it out.
+### Deploying Sample `NIMCache` and `NIMService` Resources
 
-### To Deploy a sample `NIMCache` and `NIMService` instance
+Follow the guides in the [docs](./docs) directory to deploy sample CR instances.
 
-Follow the guides [here](https://github.com/NVIDIA/k8s-nim-operator/-/tree/main/docs?ref_type=heads) to deploy sample CR instances.
+### Uninstalling the Operator
 
-### To Uninstall
 **Delete the instances (CRs) from the cluster:**
 
 ```sh
 kubectl delete -k config/samples/
 ```
 
-**Delete the APIs(CRDs) from the cluster:**
+**Delete the APIs (CRDs) from the cluster:**
 
 ```sh
 make uninstall
 ```
 
-**UnDeploy the controller from the cluster:**
+**Undeploy the controller from the cluster:**
 
 ```sh
 make undeploy
 ```
 
 ## Project Distribution
 
-Following are the steps to build the installer and distribute this project to users.
+Perform the following steps to build the installation manifests and distribute this project to users.
 
-1. Build the installer for the image built and published in the registry:
+1. Build the manifests for the image built and published in the registry:
 
-```sh
-make build-installer IMG=nvcr.io/nvidia/cloud-native/k8s-nim-operator:v0.1.0
-```
+   ```sh
+   make build-installer IMG=nvcr.io/nvidia/cloud-native/k8s-nim-operator:v0.1.0
```

Comment:
for the readme, better to replace `nvcr.io/nvidia/cloud-native/k8s-nim-operator:v0.1.0` with generic `<your-private-repository>:<tag>` as users cannot publish to NGC.